src/views/payment-sheet-views/base-paypal-view.js

```diff
@@ -10,8 +10,15 @@ var translations = require('../../translations').fiveCharacterLocales;
 var Promise = require('../../lib/promise');
 
 var ASYNC_DEPENDENCY_TIMEOUT = 30000;
-var READ_ONLY_CONFIGURATION_OPTIONS = ['offerCredit', 'locale'];
-var DEFAULT_CHECKOUTJS_LOG_LEVEL = 'warn';
+var READ_ONLY_CONFIGURATION_OPTIONS = [
+  'commit',
+  'currency',
+  'flow',
+  'intent',
+  'locale',
+  'offerCredit'
+];
+var DEFAULT_INTENT = 'authorize';
```

Comment:
Curious for my own understanding, why isn't `capture` the default intent?

Reply:
Not sure, that's just what the Pay with PayPal team set as the default when nothing is specified. I guess because it's assumed that the vault flow was default?